tests/Unit/CssInlinerTest.php

```diff
@@ -2982,6 +2982,64 @@ public function inlineCssNotInDebugModeKeepsInvalidOrUnrecognizedSelectorsInMedi
         self::assertContainsCss($css, $subject->render());
     }
 
+    /**
+     * @test
+     */
+    public function htmlWithoutHeadOrBodyElementsAndStylesCopiedToStyleBlocksIsProperlyInlined()
```

Comment:
This test should go into the tests for the base class: `AbstractHtmlProcessorTest`.

Reply:
Done